docs/src/main/asciidoc/_elytron/components/Filesystem_Security_Realm.adoc

```diff
@@ -232,3 +244,14 @@ $JBOSS_HOME/bin/elytron-tool.sh filesystem-realm -u conf/users.properties -r con
 ----
 
 This command creates new filesystem realm with users taken from users.properties file and roles taken from roles.properties file. Script `example-fs-realm.sh` that contains the commands for WildFly CLI is generated as well. The script adds this filesystem realm to the Elytron subsystem and also adds new security domain that uses this filesystem realm as a default realm.
+
+== Converting plain Filesystem realn into Encrypted Filesystem realm
+
+Since the addition of Encrypted Filesystem Realm, Elytron has added a feature to convert plain unencrypted filesystem realms, to encrypted filesystem realm. You can use `Elytron Tool` to convert these realms. Below is an example of how to run this tool from the command line:
```

Comment:
I think we could simplify this to something like:

It is possible to convert an unencrypted filesystem realm into an encrypted one using the Elytron Tool. In particular, the `filesystem-realm-encrypt` command can be used as shown below:

Reply:
Changed